examples/style_sheets/style_sheets_reference.py

```diff
@@ -65,8 +66,11 @@ def plot_colored_circles(ax, prng, nb_samples=15):
     for sty_dict, j in zip(plt.rcParams['axes.prop_cycle'], range(nb_samples)):
         ax.add_patch(plt.Circle(prng.normal(scale=3, size=2),
                                 radius=1.0, color=sty_dict['color']))
-    # Force the limits to be the same across the styles (because different
-    # styles may have different numbers of available colors).
+    ax.grid(visible=True)
+
+    # Add title for enabling grid
+    plt.title('ax.grid(True)', family='monospace', y=1.075)
```

Comment:
```suggestion
    plt.title('ax.grid(True)', family='monospace', fonstize='small')
```
Did you really need to set y by hand?

Reply:
No I suppose not. Was trying to get it looking as close to @timhoffm mockup. Thanks for the suggestion @jklymak